Breeze.UI/src/app/wallet/advanced/feedback/feedback.component.html

```diff
@@ -1,11 +1,14 @@
 <div>
     <div class="progress" *ngIf="loading">
-        <div class="progress-bar progress-bar-striped progress-bar-animated" role="progressbar" aria-valuenow="100"
-            aria-valuemin="0" aria-valuemax="100" style="width: 100%">
+        <div class="progress-bar progress-bar-striped progress-bar-animated" role="progressbar" aria-valuenow="100" aria-valuemin="0"
+            aria-valuemax="100" style="width: 100%">
             <label style="font-size:10px">Working...</label>
         </div>
     </div>
-    <div class="alert alert-danger" role="alert" *ngIf="errored">
-        <label>{{erroredText}}</label>  
+    <div *ngIf="errored">
```

Comment:
Is `ngSwitch` more appropriate here as we are showing divs based on state? https://angular.io/api/common/NgSwitch. Also would be good to get rid of inline styles and combine into common class